queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals where name like '%mon' ;
+SELECT name from animals where EXTRACT(YEAR FROM date_of_birth) between 2016 AND 2019;
+SELECT name from animals where neutered='true' AND escape_attempts>='3'; 
```

Comment:
[Required] You were expected to get animals that are neutered and escape attempts is less than 3 not more than or equal to